### ISSUE TITLE
Drop the plus sign from GTK

### ISSUE THREAD

**Issue by afranke** [dino/dino#1511]
[GTK+ got renamed to GTK four and a half years ago](https://mail.gnome.org/archives/gtk-devel-list/2019-February/msg00000.html). For a client that claims to be modern, you should update your GitHub description and the README header (possibly other places) to align with that change.

**Comment by fiaxh** [comment]
Done in b490eb662fb8fb9ddbc8bbaea2b5608e7a511158.

**Comment by afranke** [comment]
Not quite. The GitHub project description (in the sidebar on the right of https://github.com/dino/dino) still has the old text.
